### ISSUE TITLE
BottomSheetInfo (often) not rendering (fully)

### ISSUE THREAD

**Issue by ThaBrad** [DeFiCh/wallet#3139]
#### What happened:
BottomSheetInfo (sometimes) not rendering
(since ~1.5.1, maybe even down to 1.4x)

--> black screen overlay but BottomSheet not coming up (only after a while or if app went to background)

#### What you expected to happen:
--> opening of BottomSheet
![IMG_C274FCD73CAD-1](https://user-images.githubusercontent.com/18183189/178282313-def315f4-7804-44e1-a72c-7909dff4e6f1.jpeg)
![IMG_2846](https://user-images.githubusercontent.com/18183189/178282461-ab6caaa6-ced0-43dc-b919-6489ab464fa3.PNG)

#### How to reproduce it (as minimally and precisely as possible):
basically click on any info text beside oracle (world) icon

#### Anything else we need to know?:
iPhone SE(2), but also happens on iOS Simulator (iPhone 12), probably on all devices

A manual re-render didn't fix it :
```
const update = useForceUpdateHook()
useEffect(() => {
    setTimeout(() => {
      update()
      console.log('update_')
    }, 2000)
  }, [name])
```
